rental.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+require_relative './book'
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
+
+  def initialize(book, person, date)
+    @date = date
+    @person = person
+    @book = book
+  end
+  person.books << @book
+  book.rentals << @person
```

Comment:
- [ ] Why are you leaving the `belong-to` assignment outside the constructor? Did it work as intended? I'd suggest moving it inside the class constructor since it will be invoked one time on object creation.